app/Http/Controllers/Three/PostsController.php

```diff
@@ -106,16 +106,18 @@ public function store(PostRequest $request)
 
         $signup = $this->signups->get($northstarId, $request['campaign_id'], $request['campaign_run_id']);
 
-        $updating = ! is_null($signup);
+        $signupExists = ! is_null($signup);
 
-        if (! $updating) {
+        if (! $signupExists) {
             $signup = $this->signups->create($request->all(), $northstarId);
             $post = $this->posts->create($request->all(), $signup->id);
```

Comment:
Can we simplify this conditional since the `$this->posts->create` is shared between both cases?

```php
$signup = $this->signups->get($northstarId, $request['campaign_id'], $request['campaign_run_id']);

if (! $signup) {
    $signup = $this->signups->create($request->all(), $northstarId);
}

$post = $this->posts->create($request->all(), $signup->id);
```

Reply:
ah yes. nice!